Task_5/Task5.py

```diff
@@ -0,0 +1,28 @@
+# Multi paradigm programming language
+# Task 4.1, 4.2, 4.3
+# Saprykin Maksym
+# zachet_number = 16
+
+from math import *
+
+course_name = '\'Multi paradigm programming language\''
+task_number = 'Task 4.1, 4.2, 4.3 '
+first_name_student = 'Maksym'
+last_name = 'Saprykin'
+zachet_number = '16'
+
+formula = 'sin(x) * x'
+
+print(course_name + ': ' + task_number)
+print(first_name_student + ' ' + last_name + ', ' + zachet_number, '\n')
+
+# ishodna formula
+def func ():
+    f = eval(formula)
+    return f
+
+# Description Tasks
+print('1. Example:')
+print('y = ', formula)
+
+# input data
```

Comment:
looks like you even haven't started yet.
please create PRs after you finish the work